addons/website/views/snippets/snippets.xml

```diff
@@ -999,23 +999,22 @@
         </we-button-group>
     </div>
 
-    <!-- Min height of section -->
-    <div data-option-name="minHeight" data-selector="section">
+    <!-- Scroll to next section button (only for full height) -->
+    <div data-js="ScrollButton" data-selector="section">
+        <!-- Min height of section -->
         <we-button-group string="Height">
             <we-button data-select-class="" title="Fit content">Auto</we-button>
             <we-button data-select-class="o_half_screen_height" title="Half screen">50%</we-button>
-            <we-button data-select-class="o_full_screen_height" title="Full screen">100%</we-button>
+            <we-button data-select-class="o_full_screen_height" title="Full screen"  data-name="full_height_opt" data-is-full-height="true">100%</we-button>
         </we-button-group>
-    </div>
 
-    <!-- Scroll to next section button (only for full height) -->
-    <div data-js="ScrollButton" data-selector="section">
-        <we-checkbox string="Scroll down button"
+        <we-checkbox string=" ⌙ Scroll down button"
```

Comment:
@sbuhl Why this spec again? Can we drop it (again) please? If my cover is half height or is using auto height but naturally tall because of what's inside, I want to be able to add a scroll down button... (and no I don't want to enable height 100% to make the option appear).

It's just a one line option, which always make a clear button appear in the DOM... who is this confusing?

Reply:
@qsm-odoo Endless debate with you know who...

Reply:
So should I remove this functionality (@sbuhl ) ?

Reply:
Moved to https://github.com/odoo/odoo/pull/68825